UACloudLibraryServer/Models/InfoModel.cs

```diff
@@ -65,57 +67,79 @@ public AddressSpace()
             AdditionalProperties = new Tuple<string, string>[0];
         }
 
+        [Key]
+        [Column("addressspaceid")]
+        public int AddressSpaceId { get; set; }

```

Comment:
this is an old version: We already removed the IDs as part of an earlier change.